notebooks/official/CODEOWNERS

```diff
@@ -28,4 +28,5 @@
 /automl/automl_forecasting_bqml_arima_plus_comparison.ipynb @TheMichaelHu
 /automl/automl_tabular_on_vertex_pipelines.ipynb @helinwang
 /custom/custom_training_tensorboard_profiler.ipynb @itseric
```

Comment:
@ivanmkc it looks like @itseric exists but maybe doesn't have access to this repo - should we just ignore the message about the error in the CODEOWNERS file, or are you able to correct it so there's no more error?